src/main/java/com/accelad/docx4j/tags/TagUpdaterTable.java

```diff
@@ -60,12 +62,12 @@ public boolean update(Tag tag) {
     }
 
 
-    private TableRow createHeaders(String[] headers) { // NOSONAR
+    private TableRow createHeaders(TagValueTableRow headers) { // NOSONAR
         TableRow headerRow = new TableRow();
 
-        for (String header : headers) {
+        for (TagValueTableCell header : headers) {
             headerRow.addCell(TableCell.builder()
-                    .withText(header)
+                    .withText(header.toString())
```

Comment:
it will be better if TableCell.Builder could build the TableCell from a TagValueTableCell, and not from a string value.